### ISSUE TITLE
Error in permissions lesson

### ISSUE THREAD

**Issue by oC-n** [carpentries-incubator/shell-extras#78]
In the lesson on [permissions](https://carpentries-incubator.github.io/shell-extras/04-permissions/index.html), the command for removing all permissions for other users (non-owner, non-group) is given as `chmod a= final.grd`. This will instead remove all permissions from owner and group members as well as others. I believe the command should be `chmod o= final.grd` to remove permissions from *other* only.

**Comment by kynan** [comment]
Yep, agree! Want to send a fix @oC-n ? I don't have write access, so @gcapes will need to merge any pr :)

**Comment by oC-n** [comment]
Happy to sort that out, @kynan - just might take me another week or so to get to it, as racing a deadline at present.

**Comment by bagustris** [comment]
@kynan @oC-n, 

I included it in my PR #81 . Also, I change the wording from "all" to "others" since it is more appropriate.
